tests/integration-tests/tests/cfn-init/test_cfn_init/test_install_args_quotes/post_install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "post-install script has $# arguments"
+for arg in "$@"
+do
+    echo "arg: ${arg}"
+done
+
+case $# in
+    4)
```

Comment:
Is this correct?

```
$ cat test_args.sh 
#!/bin/bash

echo "$#"

$ ./test_args.sh R curl wget
3
```

Alternative we can just assert each argument individually, i.e. `"$1" == "R"`, `"$2" == "curl"` ...
It's ok to hard code our assertions in tests, to avoid potential confusion and unexpected silent failures

Reply:
Because in post_install_args, the first argument is s3 bucket url, so if we use "R curl wget", we will have 4 argument. I checked this in `cfn-init.log `.